### ISSUE TITLE
kogs: kogswrapper timeouts om rpc call sometimes, on first start or after node restart

### ISSUE THREAD

**Issue by dimxy** [dimxy/komodo#16]
increase the libnspv timeout

**Comment by dimxy** [comment]
seems this is related to another issue with blocking getaddr messages
also it was discovered libevent delays sending messages up to 5 sec (no delay on the komodo side though unless it was blocked by getaddr messages blocked in turn AddrMan lock. Repaired in issue #18
